htdocs/core/class/ldap.class.php

```diff
@@ -307,15 +307,25 @@ public function connect_bind()
 					if ($ldapdebug) {
 						dol_syslog(get_class($this)."::connect_bind serverPing true, we try ldap_connect to ".$host);
 					}
-					$this->connection = ldap_connect($host, $this->serverPort);
+					if (strnatcmp(phpversion(), '8.3.0') >= 0) {
```

Comment:
In other part of code, we are using this to compare php version
if (version_compare(phpversion(), '7.0', '>=')) {
Can you use the same method for better standardization ?
if (version_compare(phpversion(), '8.3', '>=')) {

Reply:
Thanks for tip. Code updated accordingly.